app/controllers/admin/user_management_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::UserManagementController < ApplicationController
```

Comment:
Convention dictates that you name this Admin::UsersController

Reply:
Roger that